### DISCUSSION TITLE
麻烦开发大神添加一下 小龙云盘    https://www.stariverpan.com/

### DISCUSSION THREAD

**Discussion by taoyihu** [AlistGo/alist#842] [Ideas]
麻烦开发大神添加一下 小龙云盘    https://www.stariverpan.com/

**Comment by wintopic** [comment]
附议附议！小龙云盘不限速+无限空间，值得支持！

**Comment by whws** [comment]
https://wx.stariverpan.com/web/home
这个是该网盘的链接，希望能够支持挂载。

**Comment by pipinian** [comment]
虽然不能挂载telegram这个无限存储，但如果能挂载小龙云盘也不错，听官方说数据不会丢失不用怕的
